SPECS/ndctl/ndctl.spec

```diff
@@ -59,60 +100,110 @@ developing applications that use %{name}, a library for enumerating
+
 %prep
 %setup -q ndctl-%{version}
 
 %build
-./autogen.sh
-%configure \
-    --disable-static  \
-    --enable-local    \
-    --disable-docs
-make %{?_smp_mflags}
+%meson %{?asciidoctor} %{?libtracefs} -Dversion-tag=%{version}
+%meson_build
 
 %install
-make install DESTDIR=%{buildroot}
-find %{buildroot} -name '*.la' -delete
+%meson_install
 
 %check
-make check
+%meson_test
+
+%define bashcompdir %(pkg-config --variable=completionsdir bash-completion)
 
 %post -p /sbin/ldconfig
 
 %postun -p /sbin/ldconfig
 
 %files
 %defattr(-,root,root)
-%license util/COPYING licenses/BSD-MIT licenses/CC0
+%license LICENSES/preferred/GPL-2.0 LICENSES/other/MIT LICENSES/other/CC0-1.0
 %{_bindir}/ndctl
-%{_libdir}/libndctl.so.*
-%{_datadir}/bash-completion/
-%{_sysconfdir}/ndctl/*
-%{_sysconfdir}/modprobe.d/nvdimm-security.conf
+%{_mandir}/man1/ndctl*
+%{bashcompdir}/ndctl
 %{_unitdir}/ndctl-monitor.service
+ 
+%dir %{_sysconfdir}/ndctl
+%dir %{_sysconfdir}/ndctl/keys
+%{_sysconfdir}/ndctl/keys/keys.readme
+ 
+%{_sysconfdir}/modprobe.d/nvdimm-security.conf
+ 
+%dir %{_sysconfdir}/ndctl.conf.d
+%config(noreplace) %{_sysconfdir}/ndctl.conf.d/monitor.conf
+%config(noreplace) %{_sysconfdir}/ndctl.conf.d/ndctl.conf
+
+%files -n daxctl-libs
+%doc README.md
+%license LICENSES/preferred/LGPL-2.1 LICENSES/other/MIT LICENSES/other/CC0-1.0
+%{_libdir}/libdaxctl.so.*
+	
+%files -n cxl-libs
+%doc README.md
+%license LICENSES/preferred/LGPL-2.1 LICENSES/other/MIT LICENSES/other/CC0-1.0
+%{_libdir}/libcxl.so.*
+
+	
+%files libs
+%doc README.md
+%license LICENSES/preferred/LGPL-2.1 LICENSES/other/MIT LICENSES/other/CC0-1.0
+%{_libdir}/libndctl.so.*
 
 %files devel
 %defattr(-,root,root)
-%license COPYING
+%license LICENSES/preferred/LGPL-2.1
 %{_includedir}/ndctl/
 %{_libdir}/libndctl.so
 %{_libdir}/pkgconfig/libndctl.pc
 
 %files -n daxctl
 %defattr(-,root,root)
-%license util/COPYING licenses/BSD-MIT licenses/CC0
+%license LICENSES/preferred/GPL-2.0 LICENSES/other/MIT LICENSES/other/CC0-1.0
 %{_bindir}/daxctl
-%{_libdir}/libdaxctl.so.*
-%{_datadir}/daxctl/daxctl.conf
+%{_mandir}/man1/daxctl*
+	
+%{_datadir}/daxctl
+%{bashcompdir}/daxctl
+%{_unitdir}/daxdev-reconfigure@.service
+%config %{_udevrulesdir}/90-daxctl-device.rules
+%config(noreplace) %{_sysconfdir}/daxctl.conf.d/daxctl.example.conf
+
+%files -n cxl-cli
+%license LICENSES/preferred/GPL-2.0 LICENSES/other/MIT LICENSES/other/CC0-1.0
+%{_bindir}/cxl
+%{_mandir}/man1/cxl*
+%{bashcompdir}/cxl
+%{_unitdir}/cxl-monitor.service
 
 %files -n daxctl-devel
 %defattr(-,root,root)
-%license COPYING
+%license LICENSES/preferred/LGPL-2.1
 %{_includedir}/daxctl/
 %{_libdir}/libdaxctl.so
 %{_libdir}/pkgconfig/libdaxctl.pc
 
+%files -n cxl-devel
+%license LICENSES/preferred/LGPL-2.1
+%{_includedir}/cxl/
+%{_libdir}/libcxl.so
+%{_libdir}/pkgconfig/libcxl.pc
+%{_mandir}/man3/cxl*
+%{_mandir}/man3/libcxl.3*
+
 %changelog
+*   Wed Jan 31 2024 Saul Paredes <saulparedes@microsoft.com> - 78-1
```

Comment:
It seems a github check is failing:
![image](https://github.com/microsoft/CBL-Mariner/assets/4929861/57443125-7eb3-4f49-8449-2f721acf2fad)

Reply:
Thank you @mfrw. I verified SPEC file matches https://src.fedoraproject.org/rpms/iniparser/blob/rawhide/f/iniparser.spec , follows https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/299/Spec-File-Best-Practices and fixed github check